app/users/admin.py

```diff
@@ -1,6 +1,57 @@
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin
 
+from app.users.forms import CustomUserChangeForm, CustomUserCreationForm
```

Comment:
*nitpick*

It'd be good to use a standard import format. 

Both `app.users.forms` and `.models` are in the same directory, but are imported differently.